doc/UserGuide.md

```diff
@@ -111,6 +111,10 @@ Examples:
   `viewall 1`<br>
   Views all details of the 1st person in the results of the `find` command.
 
+### Sorting address book by name : `sort`
+Sorts all entries by name in alphabetical order before displaying.
+Format: sort
+  
```

Comment:
I feel that "sort" may not fully capture what the command does, since the entries are being sorted **and** displayed. Perhaps a more descriptive name could be sortAndList or sortedList?